labeling/internal/empty.go

```diff
@@ -8,15 +8,18 @@ import (
 	"github.com/CircleCI-Public/circleci-config/labeling/labels"
 )
 
+var ignoreList = []string{
+	".",
+	"readme.md",
+	".git",
+}
+
 var EmptyRepoRules = []labels.Rule{
 	func(c codebase.Codebase, ls labels.LabelSet) (label labels.Label, err error) {
 		label.Key = labels.EmptyRepo
 		_, err = c.FindFileMatching(func(path string) bool {
-			if path == "." || strings.TrimSpace(strings.ToLower(path)) == "readme.md" {
-				return false
-			}
-
-			return true
+			path = strings.TrimSpace(strings.ToLower(path))
+			return !shouldIgnorePath(path)
 		}, "*")
```

Comment:
note there is probably a regex pattern that I can use to exclude the readme and git directory, but I opted to use simple go functions for readability.